extensions/wikia/ArticleVideo/ArticleVideo.i18n.php

```diff
@@ -5,9 +5,11 @@
 $messages['en'] = [
 	'articlevideo-watch' => 'watch',
 	'articlevideo-related-video' => 'Related video',
+	'articlevideo-feedback-question' => 'Do you like this video?',
```

Comment:
did you upload this to crowdin and filled ticket for LOC team? or created subtask for doing this?